bindings/blst.swg

```diff
@@ -350,7 +350,8 @@ import java.nio.file.*;
                                 + "/" + System.getProperty("os.arch")
                                 + "/" + libName;
     static {
-        InputStream res = $imclassname.class.getResourceAsStream(resName);
+        InputStream res = $imclassname.class.getResourceAsStream(
+                        System.getProperty("supranational.blst.jniResource", resName));
```

Comment:
Just in case, to avoid any ambiguity, suggestion is to use `$imclassname.class.getPackageName() + ".jniResource"` instead of "supranational.blst.jniResource".